draft-ietf-taps-interface.md

```diff
@@ -1378,6 +1378,10 @@ It can be used to identify send events (see {{send-events}}) related to a specif
 The optional endOfMessage parameter supports partial sending and is described in
 {{send-partial}}.
 
+Framers can be used to extend or modify the message data
+with additional information that can be processed at the receiver to detect message
```

Comment:
```suggestion
with additional information that can be processed at the receiver and to determine message
```

Reply:
Now this reads strangely - as if processing at the receiver is one thing, and determining message boundaries is another. First, what's your motivation to change it - what else would you want to do with framers at the receiver? This text could be changed to say ", e.g., to determine message boundaries at the receiver" - which would read better, but immediately raise the question: "what else?". So we'd need another example.

Reply:
I agree with Michael that the proposed changed reads a bit awkward. I'd prefer to just not change it

Reply:
The motivation was to clarify that framers can be used for other things than "just" determining message boundaries, e.g., extracting information and making them available through the message context. – it is a nit here anyway

Reply:
Then I suggest to keep the old text. We elaborate on what framers can do in the {{framing}} section that this points too anyway.